reporter/src/main/kotlin/utils/FreemarkerTemplateProcessor.kt

```diff
@@ -330,6 +330,17 @@ class FreemarkerTemplateProcessor(
         @Suppress("UNUSED") // This function is used in the templates.
         fun filterForUnresolvedVulnerabilities(vulnerabilities: List<Vulnerability>): List<Vulnerability> =
                 vulnerabilities.filter { resolutionProvider.getVulnerabilityResolutionsFor(it).isEmpty() }
+
```

Comment:
Commit message:

> to generate human-readable string

"to generate a human-readable string" or "to generate human-readable strings"